routes/w3c/groups.json

```diff
@@ -1,36 +1,370 @@
 {
-  "audio": 46884,
-  "automotive": 76043,
-  "css": 32061,
-  "das": 43696,
-  "did": 117488,
-  "dxwg": 99375,
-  "fonts": 44556,
-  "html": 115520,
-  "i18n": 32113,
-  "json-ld": 107714,
-  "media": 115198,
-  "outreach": 35532,
-  "payments": 83744,
-  "pointer-events": 59096,
-  "publishing": 100074,
-  "second-screen": 74168,
-  "service-workers": 101220,
-  "svg": 19480,
-  "testing": 49799,
-  "timed-text": 34314,
-  "tracing": 108594,
-  "vcwg": 98922,
-  "wai-apa": 83907,
-  "wai-aria": 83726,
-  "wai-gl": 35422,
-  "wasm": 101196,
-  "webapps": 114929,
-  "webappsec": 49309,
-  "webauthn": 87227,
-  "webperf": 45211,
-  "webplatform": 83482,
-  "webrtc": 47318,
-  "webxr": 109735,
-  "wot": 95969
+  "wg": {
+    "audio": 46884,
+    "automotive": 76043,
+    "css": 32061,
+    "das": 43696,
+    "did": 117488,
+    "dxwg": 99375,
+    "fonts": 44556,
+    "html": 115520,
+    "i18n": 32113,
+    "json-ld": 107714,
+    "media": 115198,
+    "outreach": 35532,
+    "payments": 83744,
+    "pointer-events": 59096,
+    "publishing": 100074,
+    "second-screen": 74168,
+    "service-workers": 101220,
+    "svg": 19480,
+    "testing": 49799,
+    "timed-text": 34314,
+    "tracing": 108594,
+    "vcwg": 98922,
+    "wai-apa": 83907,
+    "wai-aria": 83726,
+    "wai-gl": 35422,
+    "wasm": 101196,
+    "webapps": 114929,
+    "webappsec": 49309,
+    "webauthn": 87227,
+    "webperf": 45211,
+    "webplatform": 83482,
+    "webrtc": 47318,
+    "webxr": 109735,
+    "wot": 95969
+  },
+  "cg": {
+    "abcg": 118199,
+    "accessibility4children": 119612,
+    "accessibilityinindia": 61068,
+    "accesslearn": 77612,
+    "act-r": 114156,
```

Comment:
We probably don't want all of these as not all are active.

Reply:
can you filter them?

Reply:
Maybe sort them by participant count?

Reply:
let's leave them for now... alt, we ask the w3c for better data here.

Reply:
Also, "wot" is shortname for both a WG and a CG